root/components/home/Intro.js

```diff
@@ -0,0 +1,265 @@
+                        size="lg"
+                      />
+                    </button>
+
+                    <Modal onHide={this.handleClose} show={this.state.show}>
+                      <Modal.Header closeButton>
+                        <Modal.Title>{l('Scan Barcode')}</Modal.Title>
+                      </Modal.Header>
+                      <Modal.Body>
+                        {/* <BarcodeScanner */}
+                        {/*  height="100%" */}
+                        {/*  onUpdate={(err, result) => { */}
+                        {/*    if (result) { */}
+                        {/*      if (result.getText() !== this.state.data) { */}
+                        {/*        window.open('https://musicbrainz.org/search?advanced=1&type=release&query=barcode%3A' + result.getText(), '_self'); */}
+                        {/*      } */}
+                        {/*      this.setState({data: result.getText()}); */}
+                        {/*    } else { */}
+                        {/*      this.setState({data: 'Actively looking for a barcode...'}); */}
+                        {/*    } */}
+                        {/*  }} */}
+                        {/*  width="100%" */}
+                        {/* /> */}
+                      </Modal.Body>
+                      <Modal.Footer>
+                        <p>{this.state.data}</p>
+                      </Modal.Footer>
+                    </Modal>
+                  </div>
+                </div>
+                <div className="choice-chips mt-2 ms-4 mb-2">
+                  {
+                    chipData.map((data) => {
+                      if (data.key === 0) {
+                        return (
+                          <div
+                            className="chip chip--active"
+                            id={'type' + data.key}
+                            onClick={() => onChipClick(data.label)}
+                          >
+                            {data.label}
+                          </div>
+                        );
+                      }
+                      return (
+                      // eslint-disable-next-line react/jsx-key
+                        <div
+                          className="chip"
+                          id={'type' + data.key}
+                          onClick={() => onChipClick(data.label)}
+                        >
+                          {data.label}
+                        </div>
+                      );
+                    })
+                  }
+                </div>
+                <div
+                  className="row ps-2 pe-2"
+                  style={{height: '40vh', overflow: 'hidden'}}
+                >
+                  {this.props.recentAdditions.map((artwork, index) => (
+                    <ReleaseArtwork
+                      artwork={artwork}
+                      key={index}
+                    />
+                  ))}
+                </div>
+              </div>
```

Comment:
Is there a reason for the weirdly off-center, distracting background squares in test? Or is it a bug? It looks like a bug, but I'm not sure :)